gui/curs_lib.c

```diff
@@ -728,8 +728,9 @@ int mutt_do_pager(const char *banner, const char *tempfile, PagerFlags do_color,
 
   int rc;
 
-  if (!C_Pager || mutt_str_equal(C_Pager, "builtin"))
-    rc = mutt_pager(banner, tempfile, do_color, info);
+  if (!C_Pager || mutt_str_equal(C_Pager, "builtin")) {
+    rc = mutt_pager(banner, tempfile, do_color, info, NULL);
```

Comment:
`mutt_pager()` seems to account for mailbox to be `NULL` but this still feels hacky.

I don't know how to test this code path.

Reply:
This shows all the routes into the pager:

![mutt_pager](https://gist.github.com/flatcap/044ecbd2498c65ea9a85099ef317509a/raw/1faa33247d888f8e94b30ebfb95c44a2de070d52/mutt_pager.svg)

I don't think a `Context` make any sense for them.

Reply:
Help certainly works, viewing messages too. So I guess I didn't break anything